oraibridge/chain.json

```diff
@@ -0,0 +1,107 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "oraibridge",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://orai.io/",
+  "pretty_name": "OraiBridge",
+  "chain_id": "oraibridge-subnet-2",
+  "bech32_prefix": "oraib",
+  "daemon_name": "gravity",
```

Comment:
May be legitimate, but also may be leftover from working on a copy of Gravity Bridge's chain.json file?